osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -55,8 +56,11 @@ public GlobalActionContainer(OsuGameBase game)
             new KeyBinding(new[] { InputKey.Control, InputKey.Minus }, GlobalAction.DecreaseScrollSpeed),
         };
 
+        /// <summary>
+        /// Make sure that the <see cref="OsuGame"/> handles global input first, and that <see cref="BackButton"/> handles global input last.
+        /// </summary>
         protected override IEnumerable<Drawable> KeyBindingInputQueue =>
-            handler == null ? base.KeyBindingInputQueue : base.KeyBindingInputQueue.Prepend(handler);
+            (handler == null ? base.KeyBindingInputQueue : base.KeyBindingInputQueue.Prepend(handler)).OrderBy(d => d is BackButton);
```

Comment:
Instead of this, how about putting `BackButton` at the very deepest level, and proxying it to the above layer?

```
Children =
    backButton
    /* ScreenStack */
    backButton.CreateProxy()
```